src/Command.ts

```diff
@@ -134,6 +134,7 @@ export default abstract class Command {
    * @returns The original command, supports method chaining.
    */
   public async edit(interaction: CommandInteraction, message: InteractionPayload) {
+    
```

Comment:
I think some small changes accidentally got committed here and in ActionManager.ts, so we should probably keep them unchanged from the original.

Reply:
Yeah I tried playing around with those files to see how to make bot messages private but the current commit should be consistent with the main branch.